ReSharper.FSharp/test/data/features/daemon/identifierHighlighting/Functions 02.fs.gold

```diff
@@ -0,0 +1,115 @@
+﻿open |System|(0)
+
+let inline |CallInstanceProp|(1)<|'T|(2) when |'T|(3) : (member |Two|(4) : |'T|(5))> (|x|(6) : |'T|(7)) =
+    (|^T|(8) : (member |Two|(9) : |'T|(10)) (|x|(11)))
+let inline |CallInstanceMethod|(12)<|'T|(13) when |'T|(14) : (member |Two|(15) : |unit|(16) -> |'T|(17))> (|x|(18) : |'T|(19)) =
+    (|^T|(20) : (member |Two|(21) : |unit|(22) -> |'T|(23)) (|x|(24)))
+let inline |CallStaticProp|(25)<|'T|(26) when |'T|(27) : (static member |Two|(28) : |'T|(29))> =
+    (|^T|(30) : (static member |Two|(31) : |'T|(32)) ())
+let inline |CallStaticMethod|(33)<|'T|(34) when |'T|(35) : (static member |Two|(36) : |unit|(37) -> |'T|(38))> =
+    (|^T|(39) : (static member |Two|(40) : |unit|(41) -> |'T|(42)) ())
+    
+let inline |arraysAndStaticMethod|(43)<|'T|(44) when |'T|(45) : (static member |Two|(46) : |unit|(47) -> |'T|(48))> =
+    let |arr|(49) = |Array|(50).|zeroCreate|(51)<|Type|(52)> 10
+    for |i|(53) = 0 to |arr|(54).|Length|(55) do
+        |arr|(56).[|i|(57)] <- |typeof|(58)<|'T|(59)>
+    let |element|(60) = (|^T|(61) : (static member |Two|(62) : |unit|(63) -> |'T|(64)) ())
+    let |arr2|(65) = Enumerable.Repeat(element, 10) ||>|(66) |Seq|(67).|toArray|(68)
+    |arr|(69) ||>|(70) |Array|(71).|zip|(72) |arr2|(73)
+    
+let inline |arraysAndStaticProp|(74)<|'T|(75) when |'T|(76) : (static member |Two|(77) : |'T|(78))> =
+    (|^T|(79) : (static member |Two|(80) : |'T|(81)) ()) ||>|(82) |Seq|(83).|replicate|(84) 10
+    ||>|(85) |Seq|(86).|toArray|(87)
+    ||>|(88) |Array|(89).|zip|(90) ((Enumerable.Repeat(typedefof<'T>, 10)).ToArray())
```

Comment:
`((Enumerable.Repeat(typedefof<'T>, 10)).ToArray())` - for some reason auto generation did not work for this thing.
Should I manually fix this?

Reply:
There are more tests with similar bugs. Linq test is totally broken. Just realized this.

Reply:
Does adding `[<TestReferences("System", "System.Core")>]` to the test case or to `IdentifierHighlightingTest` class help? (Maybe a single reference instead of both is enough.)

Reply:
Could you please also try splitting the bigger tests here into smaller ones so it's easier to see particular things being tested?

Reply:
It helped. Thank you!